go.local.mod

```diff
@@ -6,7 +6,7 @@ replace github.com/aws-controllers-k8s/runtime => ../runtime
 
```

Comment:
FYI this is a `go.local.mod` file - these changes were automatically assumed because of the newest runtime version.